### PR TITLE
Change range(len(x)) to enumerate(x) when applicable

### PR DESCRIPTION
**Pull request by Pierre-Sassoulas**
Fix all C0200 consider-using-enumerate signaled by pylint

### CODE REVIEW

File: src_py/cursors.py [hunk 1]
```diff
@@ -365,22 +365,22 @@ def bitswap(num):
...
     cursdata = tuple(cursdata)
-
-    for line in range(len(mask)):
-        if mask[line].startswith('static char') or mask[line].startswith('static unsigned char'):
+    for line in mask:
+        if line.startswith(possible_starts):
             break
 
-    data = ' '.join(mask[line+1:]).replace('};', '').replace(',', ' ')
+    data = ' '.join(mask[i+1:]).replace('};', '').replace(',', ' ')
```
Comment (illume) [inline src_py/cursors.py:383]: I don't think this is correct to use i instead of line?

Comment (Pierre-Sassoulas) [reply src_py/cursors.py:383]: Good catch, there's a problem here, there is no enumerate(), I'm going to fix it.

Comment (Pierre-Sassoulas) [reply src_py/cursors.py:380]: I changed `for line in mask:` to ` for i, line in enumerate(mask):` so the `i` now exists.

File: src_py/cursors.py [hunk 2]
```diff
@@ -365,22 +365,19 @@ def bitswap(num):
 
     info = width, height, hotx, hoty
 
-    for line in range(len(curs)):
-        if curs[line].startswith('static char') or curs[line].startswith('static unsigned char'):
+    possible_starts = ['static char', 'static unsigned char']
+    for i, line in enumerate(curs):
+        if line.startswith(possible_starts):
```
Comment (illume) [inline src_py/cursors.py:370]: TIL: Seems startswith can't be a list, but can be a tuple.

Comment (Pierre-Sassoulas) [reply src_py/cursors.py:370]: Yeap, TIL too :) I fixed it.

File: src_py/cursors.py (hunk 1 above)
Comment (illume) [reply src_py/cursors.py:380]: cool :)

**Review by illume — APPROVED** [review]
👍 thanks.

**Pull request merged**